tests/page-objects/default/reports/reports.wdio.page.js

```diff
@@ -4,22 +4,23 @@ const utils = require('@utils');
 
 const REPORTS_LIST_ID = '#reports-list';
 const SELECT_ALL_CHECKBOX = `${REPORTS_LIST_ID} .select-all input[type="checkbox"]`;
-const reportBodyDetailsSelector = '#reports-content .report-body .details';
-const reportBodyDetails = () => $(reportBodyDetailsSelector);
-const reportTasks = () =>  $(`${reportBodyDetailsSelector} .scheduled-tasks`);
+const REPORT_BODY_DETAILS_SELECTOR = '#reports-content .report-body .details';
+const reportBodyDetails = () => $(REPORT_BODY_DETAILS_SELECTOR);
+const reportTasks = () =>  $(`${REPORT_BODY_DETAILS_SELECTOR} .scheduled-tasks`);
+const reportTargetFilter = () =>  $(`${REPORT_BODY_DETAILS_SELECTOR} [test-id="field-target-filter"]`);
```

Comment:
Are we sure there is always just one target filter?

Reply:
using field label